nautobot/dcim/templates/dcim/interface_bulk_rename.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'generic/object_bulk_rename.html' %}
+{% load helpers %}
+{% load form_helpers %}
+
+{% block title %}Renaming {{ selected_objects|length }} {{ device_name }} {{ obj_type_plural|bettertitle }}{% endblock %}
```

Comment:
Is the need for this fix exclusive to interfaces? I'd think all device-components (console ports, power ports, vminterfaces, etc. etc.) would also benefit from this. Should this be an across-the-board enhancement to `BulkRenameView` instead?